R/auto_tweet.R

```diff
@@ -107,17 +146,23 @@ action_auto_tweet <- function(twitter_token = ser_token, google_drive_auth = dri
   if (retweet_day) {
     retweet_queue <- post_tweet_of_type(retweet_queue)
   } else {
-    tweet_queue <- post_tweet_of_type(tweet_queue)
+    tweet_library_status <- post_tweet_of_type
+  }
+
+  if (tweet_library_status$restart_history) {
+    tweet_hist <- data.frame(tweet_id = tweet_library_status$just_tweeted)
+  } else {
+    tweet_hist <- data.frame(tweet_id = c(tweet_hist_list,
+                                          tweet_library_status$just_tweeted))
```

Comment:
let's put the second data.frame all on one line, e.g.

`tweet_hist <- data.frame(tweet_id = c(tweet_hist_list, tweet_library_status$just_tweeted))`

Reply:
Putting this on one line turned it into a long line. I decide to do the following:

`
tweet_hist <- data.frame(
   tweet_id = c(tweet_hist_list, tweet_library_status$just_tweeted)
)
`
Let me know if you still prefer it on one-line.